src/containers/MessageCenterContainer.tsx

```diff
@@ -0,0 +1,49 @@
+import { connect } from 'react-redux';
+
+import { MessageCenterActions } from '../actions/MessageCenterActions';
+import { MessageCenter, MessageCenterTrigger } from '../components/MessageCenter';
+
+const mapStateToPropsMC = state => {
+  return {
+    groups: state.messageCenter.groups,
+    drawerIsHidden: state.messageCenter.hidden,
+    drawerIsExpanded: state.messageCenter.expanded,
+    drawerExpandedGroupId: state.messageCenter.expandedGroupId
+  };
+};
+
+const mapDispatchToPropsMC = dispatch => {
+  return {
+    onExpandDrawer: () => dispatch(MessageCenterActions.togleExpandedMessageCenter()),
+    onHideDrawer: () => dispatch(MessageCenterActions.hideMessageCenter()),
+    onToggleGroup: group => dispatch(MessageCenterActions.toggleGroup(group.id)),
+    onMarkGroupAsRead: group => dispatch(MessageCenterActions.markGroupAsRead(group.id)),
+    onClearGroup: group => dispatch(MessageCenterActions.clearGroup(group.id)),
+    onNotificationClick: message => dispatch(MessageCenterActions.markAsRead(message.id)),
+    onDismissNotification: () => console.log('dismiss')
+  };
+};
+
+const mapStateToPropsMCT = state => {
```

Comment:
What does MCT stand for? Might as well spell it out. It will get compiled out at production so there is no penalty for it.

Reply:
MessageCenterTrigger, I can be more verbose sure.